### PR TITLE
DV2 : Speedseq Sv

### PR DESCRIPTION
**Pull request by jasonwalker80**
A few changes to allow the SpeedseqSv DV2 module to run within a build.

### CODE REVIEW

**Comment by tmooney** [comment]
+1 🏊

**Comment by jasonwalker80** [comment]
@tmooney I had to add two more commits to this PR to get a successful build. could you please review these additional commits?

File: lib/perl/Genome/Model/Tools/DetectVariants2/SpeedseqSv.pm [hunk 1]
```diff
@@ -54,6 +54,12 @@ sub _detect_variants {
 
     my $set = Genome::Model::Tools::Speedseq::Sv->create(%final_cmd);
     $set->execute();
+
+    # Make a symlink to the actual VCF as the DV2 Dispatcher expects an output of svs.hq rather than BED or VCF output
+    my ($vcf_file) = glob($self->_sv_staging_output .'*.vcf.gz');
+    if ($vcf_file) {
+        symlink($vcf_file,$self->_sv_staging_output);
+    }
```
Comment (tmooney) [inline lib/perl/Genome/Model/Tools/DetectVariants2/SpeedseqSv.pm]: This could be in `sub _generate_standard_files` instead of tacked on the end here.

**Comment by tmooney** [comment]
The new changes seem fine to me!  Up to you if you want to move the symlinking.

**Comment by susannasiebert** [comment]
+1

**Comment by jasonwalker80** [comment]
I think I'll leave the symlinking for now.  Manta is another potential SV detector to implement.  I suspect each detector will have a different output file type and name.

**Pull request merged**